src/components/core/badge/badge.test.tsx

```diff
@@ -0,0 +1,94 @@
+        expect(icon).toBeFalsy();
+    });
+
+    it('Should render only the icon', () => {
+        const { container } = render(
+            <Badge visiblity="icon-only" icon_src={CheckIconSVG}>
+                Badge Text Content
+            </Badge>,
+        );
+        const icon = container.querySelector('img') as HTMLImageElement;
+        expect(icon).toBeInTheDocument();
+        expect(container.textContent).toBeFalsy();
+    });
+
+    it('Should render small badge', () => {
+        const { container } = render(<Badge size="small">Badge Text Content</Badge>);
+        const badgeContainer = container.querySelector('span');
+        expect(badgeContainer).toHaveAttribute('class', expect.stringContaining('size-small'));
+    });
+
+    it('Should render medium badge', () => {
+        const { container } = render(<Badge size="medium">Badge Text Content</Badge>);
+        const badgeContainer = container.querySelector('span');
+        expect(badgeContainer).toHaveAttribute('class', expect.stringContaining('size-medium'));
+    });
+
+    it('Should render large badge', () => {
+        const { container } = render(<Badge size="large">Badge Text Content</Badge>);
+        const badgeContainer = container.querySelector('span');
+        expect(badgeContainer).toHaveAttribute('class', expect.stringContaining('size-large'));
+    });
+
+    it('Should render badge with regular font weight', () => {
+        const { container } = render(<Badge label="regular">Badge Text Content</Badge>);
+        const badgeContainer = container.querySelector('span');
+        expect(badgeContainer).toHaveAttribute('class', expect.stringContaining('label-regular'));
+    });
+
+    it('Should render badge with bold font weight', () => {
+        const { container } = render(<Badge label="bold">Badge Text Content</Badge>);
+        const badgeContainer = container.querySelector('span');
+        expect(badgeContainer).toHaveAttribute('class', expect.stringContaining('label-bold'));
+    });
+
+    it('Should render badge with tight spacing', () => {
+        const { container } = render(<Badge spacing="tight">Badge Text Content</Badge>);
+        const badgeContainer = container.querySelector('span');
+        expect(badgeContainer).toHaveAttribute('class', expect.stringContaining('padding-tight'));
+    });
+
+    it('Should render badge with loose spacing', () => {
+        const { container } = render(<Badge spacing="loose">Badge Text Content</Badge>);
+        const badgeContainer = container.querySelector('span');
+        expect(badgeContainer).toHaveAttribute('class', expect.stringContaining('padding-loose'));
+    });
+
+    it('Should match with snapshot', () => {
+        const { container } = render(
+            <Badge visiblity="icon-and-label" icon_src={CheckIconSVG}>
+                Badge Text Content
+            </Badge>,
+        );
+        expect(container).toMatchSnapshot();
```

Comment:
```suggestion
```
We don't plan to do any snapshot testing for deriv-ui for now. Lets remove these tests

Reply:
removed the snapshot testings :pray: